man/rhsm.conf.5

```diff
@@ -221,6 +221,13 @@ inotify
 .RS 4
 Inotify is used for monitoring changes in directories with certificates. Currently only the /etc/pki/consumer directory is monitored by the rhsm.service. When this directory is mounted using a network file system without inotify notification support (e.g. NFS), then disabling inotify is strongly recommended. When inotify is disabled, periodical directory polling is used instead.
 .RE
+.PP
+no_progress_messages
+.RS 4
+Set to
+\fI1\fR
+disable progress reporting. subscription-manager writes temporary informational messages to the standard output when it is fetching certificates or updating user information. This feature may not be desired in some situations, changing this option prevents those messages from being displayed.
```

Comment:
Isn't comma missing before "when"?

Reply:
I wouldn't use it there, personally, but we'd have to ask native speaker for that. I'll try to phrase it a bit differently so the important part (when it is...) is not at the end of the sentence.